projects/didcomm-mediator.md

```diff
@@ -0,0 +1,104 @@
+4. **Open-Source Collaboration and Innovation**: As an open-source project, DIDComm Mediator is developed and maintained by a global community.
+5. **Security and Privacy**: The service supports secure DIDComm messaging, ensuring that communication between agents is encrypted and privacy-preserving. A DIDComm mediator cannot decrypt the contents of the message sent to a client agent. This is critical in protecting user data and maintaining the security of decentralized identity systems.
+
+## Code of Conduct
+
+ACA-Py follows the [Hyperledger Code of Conduct](https://github.com/hyperledger/aries-agent-test-harness/blob/main/CODE_OF_CONDUCT.md), which is what the OWF code of conduct is based on.
+
+## TAC Sponsor
+
+* Tracy Kuhrt
+* TBD
+
+## Project License
+
+The project has an Apache 2.0 license: [Aries Mediator Service - License](https://github.com/hyperledger/aries-mediator-service/blob/main/LICENSE)
+
+## Source Control
+
+The project is hosted in GitHub and includes the following repositories:
+
+* [Aries Mediator Service](https://github.com/hyperledger/aries-mediator-service)
+* [Aries SocketDock](https://github.com/hyperledger/aries-socketdock)
+
+## Issue Tracker
+
+Issues are tracked using GitHub's Issues feature in the corresponding repository.
+
+## External Dependencies
+
+DIDComm Mediator Service is deployed on an DIDComm Agent framework with examples
+included using [ACA-Py](https://aca-py.org) and
+[Credo-TS](https://github.com/openwallet-foundation/credo-ts). DIDComm Mediator
+Service also uses (via its embedded DIDComm agent)
+[Askar](https://github.com/hyperledger/aries-askar) for key management services
+and secure storage.
+
+## Release Methodology
+
+DIDComm Mediator Service uses GitHub Actions for dependency notifications and
+management. The primary artifacts are Docker files that can be configured and
+used to deploy a fully featured, out of the box DIDComm mediator.
+
+## Initial Maintainers
+
+| Name           | Github      | Organization                             |
+| -------------- | ----------- | ---------------------------------------- |
+| Stephen Curran | swcurran    | Funded by Government of British Columbia |
+| Wade Barnes    | WadeBarnes  | Funded by Government of British Columbia |
+| Daniel Bluhm   | dbluhm      | Indicio, PBC                             |
+| Timo Glastra   | TimoGlastra | Animo Solutions                          |
+
+## Proposed Project Governance
+
+The current governance model under Hyperledger is consensus-based. This means that decisions are made through discussions, with the aim of community consensus, as outlined in the [Aries Project Charter](https://docs.google.com/document/d/1F6RbR7xDaBt5CDJhqLJzR4c1pDJtyPGshp9fy6eVtSM/edit?usp=sharing). In cases where no clear consensus is established, a project Technical Steering Committee, or the maintainers (those with escalated GitHub privileges) are granted a louder voice. This approach has proven effective.
```

Comment:
Is the intention for this project to be distinct from the other Aries proposals and their governance processes when it moves to OWF?

Reply:
DIDComm Mediator Service is proposed to be a separate project at OWF, independent of the other artifacts stemming from the existing Hyperledger Aries project, with separate governance. That reflects the distinct purpose of the project, and community around it.